spec/models/bridge_spec.rb

```diff
@@ -1,56 +1,59 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require './spec/support/main_helper'
```

Comment:
You are requring this in every file so it's best if this require went into rails_helper. That's what it's there for!